src/config.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
     blockTypeLimits: {},
     required: [],
     uploadUrl: 'http://api.letudiant.lk/edt/media/upload',
+    apiUrl: 'http://api.letudiant.lk/edt',
```

Comment:
This should not be here; as I showed you, it should be passed as a param in the SirTrevor.init function